api/src/main/java/org/apache/cloudstack/api/response/DiskOfferingResponse.java

```diff
@@ -155,6 +155,10 @@
     @Param(description = "the vsphere storage policy tagged to the disk offering in case of VMware", since = "4.15")
     private String vsphereStoragePolicy;
 
+    @SerializedName("disksizestrictness")
```

Comment:
```suggestion
    @SerializedName("ApiConstants.DISK_SIZE_STRICTNESS")
```

Can use the defined constant.